profile_library/shelly/manufacturer.json

```
@@ -1,4 +1,6 @@
 {
-  "name": "shelly",
-  "aliases": []
+  "name": "Shelly",
```

```suggestion
  "name": "shelly",
```
Please keep this lowercase for now, as I remember there was a issue with case sensitivity making profile not discovered correctly when it is not lowercase.
Also all manufacturers listed in manufacturer dropdown in config flow and https://library.powercalc.nl are lowercase, so wan't to keep that consistent as well.
I agree we can approve on that. Might revisit in the near future, but for now please keep it this way.